tests/libtest.sh

```diff
@@ -21,8 +21,8 @@ lv_exists() {
 }
 
 clean_config_files() {
-  rm -f /etc/sysconfig/docker-storage-setup
-  rm -f /etc/sysconfig/docker-storage
+  rm -f $1
+  rm -f $2
```

Comment:
Is this function really useful anymore as a library funciton?

Given file names are being passed in by callers, it might be simpler if callers do "rm -f INPUTFILE OUTPUTFILE"